### PR TITLE
Backport locales from master

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Review by KN4CK3R — APPROVED** [review]

File: options/locale/locale_pt-PT.ini [hunk 1]
```diff
@@ -1058,6 +1060,10 @@ editor.add_tmpl=Adicionar '<filename>'
 editor.add=Adicionar '%s'
 editor.update=Modificar '%s'
 editor.delete=Eliminar '%s'
+editor.patch=Aplicar remendo (patch)
+editor.patching=Remendando (patching):
+editor.fail_to_apply_patch=Não foi possível aplicar o remendo (patch) '%s'
+editor.new_patch=Novo remendo (patch)
```
Comment (6543) [inline options/locale/locale_pt-PT.ini]:
```suggestion
```

File: options/locale/locale_de-DE.ini [hunk 2]
```diff
@@ -1105,6 +1107,7 @@ commits.signed_by_untrusted_user_unmatched=Signiert von nicht vertrauenswürdige
 commits.gpg_key_id=GPG-Schlüssel-ID
 commits.ssh_key_fingerprint=SSH-Key-Fingerabdruck
 
+
```
Comment (6543) [inline options/locale/locale_de-DE.ini]:
```suggestion
```

File: options/locale/locale_pt-PT.ini [hunk 3]
```diff
@@ -1113,6 +1121,7 @@ commits.signed_by_untrusted_user_unmatched=Assinado por um utilizador não fiáv
 commits.gpg_key_id=ID da chave GPG
 commits.ssh_key_fingerprint=Identificação digital da chave SSH
 
+
```
Comment (6543) [inline options/locale/locale_pt-PT.ini]:
```suggestion
```

File: options/locale/locale_pt-PT.ini [hunk 4]
```diff
@@ -1093,6 +1099,8 @@ editor.cannot_commit_to_protected_branch=Não é possível cometer para o ramo p
 editor.no_commit_to_branch=Não é possível cometer imediatamente para o ramo porque:
 editor.user_no_push_to_branch=O utilizador não pode enviar para o ramo
 editor.require_signed_commit=O ramo requer um cometimento assinado
+editor.cherry_pick=Escolher a dedo %s para:
+editor.revert=Reverter %s para:
```
Comment (6543) [inline options/locale/locale_pt-PT.ini]:
```suggestion
```

File: options/locale/locale_ru-RU.ini [hunk 5]
```diff
@@ -2146,6 +2147,7 @@ branch.new_branch_from=Создать новую ветку из '%s'
 branch.renamed=Ветка %s была переименована в %s.
 
 tag.create_tag=Создать тег <strong>%s</strong>
+
```
Comment (6543) [inline options/locale/locale_ru-RU.ini]:
```suggestion
```

File: options/locale/locale_ru-RU.ini [hunk 6]
```diff
@@ -1080,6 +1080,7 @@ commits.signed_by_untrusted_user=Подписано ненадежным пол
 commits.signed_by_untrusted_user_unmatched=Подписан ненадежным пользователем, который не соответствует коммиту
 commits.gpg_key_id=Идентификатор GPG ключа
 
+
```
Comment (6543) [inline options/locale/locale_ru-RU.ini]:
```suggestion
```

File: options/locale/locale_uk-UA.ini [hunk 7]
```diff
@@ -2111,6 +2117,7 @@ branch.new_branch_from=Створити нову гілку з '%s'
 branch.renamed=Гілку %s перейменовано на %s.
 
 tag.create_tag=Створити тег <strong>%s</strong>
+
```
Comment (6543) [inline options/locale/locale_uk-UA.ini]:
```suggestion
```

File: options/locale/locale_zh-CN.ini [hunk 8]
```diff
@@ -1058,6 +1060,10 @@ editor.add_tmpl=添加 '<filename>'
 editor.add=添加 '%s'
 editor.update=更新 '%s'
 editor.delete=删除 '%s'
+editor.patch=应用补丁
+editor.patching=打补丁：
+editor.fail_to_apply_patch=无法应用补丁'%s'
+editor.new_patch=新补丁
```
Comment (6543) [inline options/locale/locale_zh-CN.ini]:
```suggestion
```

File: options/locale/locale_zh-CN.ini [hunk 9]
```diff
@@ -1113,6 +1121,7 @@ commits.signed_by_untrusted_user_unmatched=由与提交者不匹配的未授信
 commits.gpg_key_id=GPG 密钥 ID
 commits.ssh_key_fingerprint=SSH 密钥指纹
 
+
```
Comment (6543) [inline options/locale/locale_zh-CN.ini]:
```suggestion
```

File: options/locale/locale_zh-CN.ini [hunk 10]
```diff
@@ -1093,6 +1099,8 @@ editor.cannot_commit_to_protected_branch=不可以提交到受保护的分支 '%
 editor.no_commit_to_branch=无法直接提交分支，因为：
 editor.user_no_push_to_branch=用户不能推送到分支
 editor.require_signed_commit=分支需要签名提交
+editor.cherry_pick=Cherry-pick %s 到：
+editor.revert=将 %s 还原到：
```
Comment (6543) [inline options/locale/locale_zh-CN.ini]:
```suggestion
```

File: options/locale/locale_zh-TW.ini [hunk 11]
```diff
@@ -1073,6 +1099,8 @@ editor.cannot_commit_to_protected_branch=無法提交到受保護的分支「%s
 editor.no_commit_to_branch=無法直接提交到分支因為：
 editor.user_no_push_to_branch=使用者無法推送到分支
 editor.require_signed_commit=分支僅接受經簽署的提交
+editor.cherry_pick=Cherry-pick %s 到：
+editor.revert=還原 %s 到：
```
Comment (6543) [inline options/locale/locale_zh-TW.ini]:
```suggestion
```

File: options/locale/locale_zh-TW.ini [hunk 12]
```diff
@@ -1038,6 +1060,10 @@ editor.add_tmpl=新增「<filename>」
 editor.add=新增「%s」
 editor.update=更新「%s」
 editor.delete=刪除「%s」
+editor.patch=套用 Patch
+editor.patching=正在 Patch:
+editor.fail_to_apply_patch=無法套用 Patch「%s」
+editor.new_patch=新增 Patch
```
Comment (6543) [inline options/locale/locale_zh-TW.ini]:
```suggestion
```

File: options/locale/locale_zh-TW.ini [hunk 13]
```diff
@@ -1093,6 +1121,7 @@ commits.signed_by_untrusted_user_unmatched=由不受信任且與提交者不相
 commits.gpg_key_id=GPG 金鑰 ID
 commits.ssh_key_fingerprint=SSH 金鑰指紋
 
+
```
Comment (6543) [inline options/locale/locale_zh-TW.ini]:
```suggestion
```

File: options/locale/locale_pt-PT.ini [hunk 14]
```diff
@@ -549,6 +549,8 @@ continue=Continuar
 cancel=Cancelar
 language=Idioma
 ui=Tema
+hidden_comment_types=Tipos de comentários ocultos
+saved_successfully=As suas configurações foram guardadas com sucesso.
```
Comment (6543) [inline options/locale/locale_pt-PT.ini]:
```suggestion
```

File: options/locale/locale_zh-CN.ini [hunk 15]
```diff
@@ -549,6 +549,8 @@ continue=继续操作
 cancel=取消操作
 language=界面语言
 ui=主题
+hidden_comment_types=隐藏的评论类型
+saved_successfully=您的设置已成功保存。
```
Comment (6543) [inline options/locale/locale_zh-CN.ini]:
```suggestion
```

File: options/locale/locale_zh-TW.ini [hunk 16]
```diff
@@ -530,10 +545,12 @@ update_profile_success=已更新您的個人資料。
 change_username=您的帳號已更改。
 change_username_prompt=注意：修改帳號也會更改您的帳戶的 URL。
 change_username_redirect_prompt=舊的帳號被領用前，會重新導向您的新帳號。
-continue=繼續操作
+continue=繼續
 cancel=取消
 language=語言
 ui=佈景主題
+hidden_comment_types=隱藏的留言類型
+saved_successfully=您的設定已成功儲存。
```
Comment (6543) [inline options/locale/locale_zh-TW.ini]:
```suggestion
```

**Review by 6543 — COMMENTED** [review]
ok all new/refactored strings from master are removed ...

so most important this will backport all the webauthn strings :)

**Review by Gusted — APPROVED** [review]

**Pull request merged**